man/systemd.unit.xml

```diff
@@ -619,7 +619,7 @@
       <varlistentry>
         <term><varname>Requires=</varname></term>
 
-        <listitem><para>Similar to <varname>Wants=</varname>, but declares a stronger
+        <listitem><para>Similar to <varname>Wants=</varname>, but declares a stronger requirement
```

Comment:
nit: "declares a stronger requirement dependency" uses singular article with two nouns and sounds a bit off. Maybe "declares stronger requirement and dependency" ?

Reply:
or maybe "declares a stronger, required dependency" ?

Reply:
hmm, we talk about "requirement dependency" as opposed to "ordering dependency" all the time, as the two categories of deps we have (requirement dep is Requires=/Wants=/Requisite=/… and ordering deps are Before=/After=). I'd stick to this naming I must say. Or at least if we change it we should change it across the board, in all man pages, with better terms for these two categories of dependencies, but I couldn't come up with anything better...

Reply:
ah that's fine then, didn't realise it was a common idyom in the docs, consistency is more important